doc/dev/release.md

```diff
@@ -182,6 +176,7 @@ Commit changes to the following six files:
 - `internal/pkg/scaffold/ansible/gopkgtoml.go`: same as for `internal/pkg/scaffold/gopkgtoml.go`.
 - `internal/pkg/scaffold/helm/gopkgtoml.go`: same as for `internal/pkg/scaffold/gopkgtoml.go`.
 - `CHANGELOG.md`: update the `## Unreleased` header to `## v1.3.0`.
+- `doc/user/install-operator-sdk.md`: update the linux and macOS URLs to point to the new release URLs.
```

Comment:
Actually one more thing. Can you update the top-level `release.sh` script to verify these version changes, as per the description in this file in the next step (line 185):

> The top-level `release.sh` script will take care of verifying versions in files described in step 1...

Reply:
WDYT we make user set the variable so we only change it one place? Otherwise, we need to change it and later verify the change in the script in multiple places.

Reply:
SGTM

Reply:
@joelanford Done, PTAL, thanks!